numpy/core/tests/test_numeric.py

```diff
@@ -2207,11 +2207,19 @@ def test_broadcast_in_args(self):
             for a, ia in zip(arrs, mit.iters):
                 assert_(a is ia.base)
 
+        # gh-6899
```

Comment:
I think this deserves its own test method, `test_broacast_single_arg` or something along that line.